### PR TITLE
Docs: added FLRW cosmology cheat sheet

### LINKED ISSUE #4244: list of cosmology functions in documentation
In a recent conversation, someone pointed that it would be useful to have a quick list reference of the cosmology functions that are available in the package.   A table with each function, link to the function, and brief description of what it does would be very helpful in the docs.

### PR DESCRIPTION
**Pull request by KING-SID**
Closes #4244

Created a quick reference document of all the FLRW cosmology methods which links back to the main documentation if more information is required.

### CODE REVIEW

**Comment by KING-SID** [comment]
It seems that some of the checks have failed. Could I get help on clarifying what the issue was? All I did was add the file and from what I see when I click details and look at the error out but, it doesn't seem to be anything related to the file I created. Any advice and feedback would be greatly appreciated

**Comment by crawfordsm** [comment]
Thanks for the contribution!  

* The first error does not seem related to your pull request but potentially due to a time out.  The test likely needs to be restarted.
* The second error (circle-ci: html-docs) looks to be related to your pull request.  I think you do need to reference this information in the main narrative docs in the index.rst file in the same directory.

**Comment by KING-SID** [comment]
@crawfordsm How do I reference the file? I am not really sure how I am meant to solve this and so your guidance would be greatly appreciated

**Comment by KING-SID** [comment]
I added the `:orphan:` line at the beginning of the document to remove the warning/error of the file not being added to any toctree. I did this because currently thhere doesn't seem to be a clear idea on where to place the document. 

After committing it and having it run the tests, it samed to have failed the same test once again.
Can I kindly be guided on how to understand the error so that I may act on it and fix the issue appropriately?

**Comment by crawfordsm** [comment]
I'm not sure if it will fix it, but adding a sentence at the end of the the introduction to link to this I think would help make this page more accessible.   Something along the lines of: 
   
    The FLRW Cosmology Method Cheat Sheet provides a quick reference to the methods that are available.   

The page should be linked to where it is called.

File: docs/cosmology/FLRWCheatSheet.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
+:orphan:
+
+==================================
+FLRW Cosmology Method Cheat Sheet
+==================================
+ A class describing an isotropic and homogeneous (Friedmann-Lemaitre-Robertson-Walker) cosmology.
+
+ Refer to `FLRW documentation for detailed information <http://docs.astropy.org/en/latest/api/astropy.cosmology.FLRW.html/>`_.
+
```
Comment (crawfordsm) [inline docs/cosmology/FLRWCheatSheet.rst]: These should links to the classes or the pages and not links to the online documentation.  Please see some of the other documentation pages for examples for how this should be linked.

Comment (crawfordsm) [reply docs/cosmology/FLRWCheatSheet.rst]: For example

    `~astropy.cosmology.FLRW`

rather than the link

Comment (KING-SID) [reply docs/cosmology/FLRWCheatSheet.rst]: Thanks, I have changed the link to the appropriate format

**Comment by KING-SID** [comment]
> I'm not sure if it will fix it, but adding a sentence at the end of the the introduction to link to this I think would help make this page more accessible. Something along the lines of:
> 
> ```
> The FLRW Cosmology Method Cheat Sheet provides a quick reference to the methods that are available.   
> ```
> The page should be linked to where it is called.

How do I determine what the link to this page is?

**Comment by pllim** [comment]
`intersphinx` will take care of generating the real URL, you don't have to worry about it.

**Comment by pllim** [comment]
Unfortunately, I don't think this quite cut it albeit a good start. Changes required:

1. Not make this an orphaned page but link it from somewhere.
2. Use proper intersphinx notation.

**Pull request closed without merging**